examples/calendar/backend/privacy/verification_key.json

```diff
@@ -0,0 +1,94 @@
+{
+ "protocol": "groth16",
+ "curve": "bn128",
```

Comment:
I am not sure you can verify bn128 onchain, only BN254. I might be forgetting things though. Please, check

Reply:
I was able to verify it on chain using groth16 and bn128.